docs/development_environment_set_up.md

```diff
@@ -0,0 +1,98 @@
+### Initial set up - Getting started
```

Comment:
So this is an outdated "getting started"?  that gets referred to in the new getting started as the "troubleshooting" section - should it be relabelled simply "troubleshooting", and does it even make sense to have a troubleshooting section?  

My concern is that it's not clear how many will actually read a trouble shooting section.  Some users will ask in slack, others will just search google (with mixed results) and others will give up.  My concern in particular is that the troubleshooting suggestions go out of date and we don't have mechanisms like SO does to keep things current.

I guess if we are consistent in always pointing everyone to the trouble shooting section when they ask in slack, and if we can link out to SO rather than copy pasting in solutions we found there we might be able to keep things under control ...